src/openzaak/components/zaken/openapi.yaml

```diff
@@ -493,7 +493,7 @@ paths:
           format: uri
       - name: resultaattype
         in: query
-        description: URL-referentie naar het RESULTAATTYPE (in de Catalogi API).
+        description: URL naar de/het gerelateerde resultaattype
```

Comment:
Please change this back -- not mentioning this for all URL-fields.

Reply:
done